docs/reference/contrib/searchpromotions.md

```diff
@@ -10,6 +10,8 @@ The `searchpromotions` module provides the models and user interface for managin
 
 "Promoted search results" allow editors to explicitly link relevant content to search terms, so results pages can contain curated content in addition to results from the search engine.
 
+As of Wagtail 5.0, the `searchpromotions` module contains the `Query` and `QueryDailyHits` models used to log search queries.
```

Comment:
```suggestion
    ```{versionadded} 5.0
    The `searchpromotions` module contains the `Query` and `QueryDailyHits` models used to log search queries.
    ```
```

Just a drive-by - we can use the built in admonitions for this kind of thing.

Reply:
I don't think this paragraph is necessary, the first paragraph in the added "Managing stored search queries" section below should be sufficient.